libparsec/crates/client/src/workspace/merge.rs

```diff
@@ -147,11 +147,34 @@ pub(super) fn merge_local_file_manifest(
 pub(super) fn merge_local_folder_manifest(
     local_author: DeviceID,
     timestamp: DateTime,
+    prevent_sync_pattern: Option<&Regex>,
     local: &LocalFolderManifest,
     remote: FolderManifest,
 ) -> MergeLocalFolderManifestOutcome {
+    // Destruct local and remote manifests to ensure this code with fail to compile
+    let LocalFolderManifest {
+        base:
+            FolderManifest {
+                id: base_id,
```

Comment:
the use of `local_` prefix instead of just `base_xxx` is important since we also have a base provided by `local_from_remote.base`

Reply:
done